src/kibana/components/agg_types/buckets/date_histogram.js

```diff
@@ -30,8 +30,17 @@ define(function (require) {
       },
       makeLabel: function (agg) {
         var output = this.params.write(agg);
-        var params = output.params;
-        return params.field + ' per ' + (output.metricScaleText || output.bucketInterval.description);
+        var interval = output.metricScaleText || output.bucketInterval.description;
+        return output.params.field + ' per ' + interval;
+      },
+      makeScaleLabel: function (agg) {
+        var output = this.params.write(agg);
```

Comment:
I'm not a fan of the `#makeLabel()` and `#makeScaleLabel()` separation. Since we want most labels in Kibana to provide info about the scaling, I'm thinking that `#makeLabel()` should return the whole thing.

Unfortunately, the label will regularly be of epic length and cause the vis editor sidebar to wrap oddly. To prevent this, I think we should:
  1. accept an arg so that `#makeLabel()` will return a shorter string.
  2. add a `#makeConciseLabel()` method to `AggType` and `AggConfig` which simply calls `#makeLabel()` by default

I'm inclined to go with 2. since it keeps the signature of `#makeLabel()` the same across all types, and the only non-date_histogram change would be [here](https://github.com/elastic/kibana/blob/60d05b39e5ad04f9ee55ae4c9698ed02d84c9c8b/src/kibana/plugins/visualize/editor/agg.js#L35-L36)